### PR TITLE
bridge_ui: MetaMask wallet_addEthereumChain support

### PR DESCRIPTION
**Pull request by kev1n-peters**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/1085)

### CODE REVIEW

File: bridge_ui/src/utils/metaMaskChainParameters.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    nativeCurrency: { name: "Fantom", symbol: "FTM", decimals: 18 },
+    rpcUrls: ["https://rpc.ftm.tools"],
+    blockExplorerUrls: ["https://ftmscan.com"],
+  },
+  686: {
+    chainId: "0x2ae",
+    chainName: "Karura Network",
+    nativeCurrency: { name: "Karura Token", symbol: "KAR", decimals: 18 },
+    // TODO: add rpcUrl
+    rpcUrls: [],
```
Comment (kev1n-peters) [inline bridge_ui/src/utils/metaMaskChainParameters.ts:81]: Couldn't find any valid Karura and Acalla RPC urls, so these will fail to add. Even chainlist.org doesn't support adding them.

File: bridge_ui/src/utils/metaMaskChainParameters.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    chainName: "Ethereum Mainnet",
+    nativeCurrency: { name: "Ether", symbol: "ETH", decimals: 18 },
+    rpcUrls: ["https://rpc.ankr.com/eth"],
+    blockExplorerUrls: ["https://etherscan.io"],
+  },
+  3: {
+    chainId: "0x3",
+    chainName: "Ropsten",
+    nativeCurrency: { name: "Ropsten Ether", symbol: "ROP", decimals: 18 },
+    rpcUrls: ["https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161"],
```
Comment (kev1n-peters) [inline bridge_ui/src/utils/metaMaskChainParameters.ts:30]: The infura rpcUrls for Ropsten and Gorli are from here: https://rpc.info/

Comment (kev1n-peters) [inline bridge_ui/src/utils/metaMaskChainParameters.ts:30]: The infura rpcUrls for Ropsten and Gorli are from here: https://rpc.info/

File: bridge_ui/src/utils/metaMaskChainParameters.ts (hunk 1 above)
Comment (evan-gray) [reply bridge_ui/src/utils/metaMaskChainParameters.ts:81]: Updating :)

**Review by evan-gray — APPROVED** [review]

**Review by chase-45 — APPROVED** [review]

**Pull request merged**